eth/backend.go

```diff
@@ -376,9 +376,9 @@ func (s *Ethereum) APIs() []rpc.API {
 	apis = append(apis, s.engine.APIs(s.BlockChain())...)
 
 	// Append any Sequencer APIs as enabled
-	if s.config.RollupSequencerEnableTxConditional {
+	if s.config.RollupSequencerTxConditionalEnabled {
 		log.Info("Enabling eth_sendRawTransactionConditional endpoint support")
-		apis = append(apis, sequencerapi.GetSendRawTxConditionalAPI(s.APIBackend, s.seqRPCService))
+		apis = append(apis, sequencerapi.GetSendRawTxConditionalAPI(s.APIBackend, s.seqRPCService, s.config.RollupSequencerTxConditionalRateLimit))
```

Comment:
Can you convert to `rate.Limit` here, because after this the unit is lost. rate.Limit implies "items/second" so it's still useful.